gradle/distribution.gradle

```diff
@@ -51,6 +51,18 @@ shadowJar {
     mergeServiceFiles()
 }
 
+task generateVersionFile {
+    doLast {
+        def resourceDir = sourceSets.main.output.resourcesDir
+        def versionFile = new File(resourceDir, "META-INF/neonbee/neonbee-version.txt")
+        versionFile.text = project.version.replace('-SNAPSHOT', '')
```

Comment:
Could it be beneficial if we don't loose the snapshot information? @s4heid What do you think?

Reply:
I think the `-SNAPSHOT` version is not very informative, because snapshot releases are not definite. It would be rather interesting to see the git commit hash, instead.

At the end you are right, @pk-work, [snapshot releases](https://github.com/SAP/neonbee/blob/4f0bfaeabeb58f5efc8ba2c0829642b7dd6a99fa/.github/workflows/publish.yml#L240) are published are published to maven central and it would be weird if they contain a non snapshot version. So, better keep the suffix.

Reply:
Okay, then lets keep the suffix. @wowselim is this also fine for you?

Reply:
done!

Reply:
Thank you very much for your contribution and your patience.